android/src/main/java/mediabrowser/apiinteraction/android/AndroidLogger.java

```diff
@@ -0,0 +1,51 @@
+package mediabrowser.apiinteraction.android;
```

Comment:
can we start transitioning to `jellyfin` package?

Reply:
That should probably be done all at once here or it would be very weird. Some packages would be in one folder and others in a different folder.

Reply:
I would like to do that all at once. It really bugs me that they didn’t follow standard namespace conventions i.e. `org.jellyfin.package`. At the same time I plan to divide the project into two gradle modules so we can have a proper Java client without the Android dependencies.